packages/expo-notifications/CHANGELOG.md

```diff
@@ -128,11 +129,9 @@ _This version does not introduce any user-facing changes._
 - Changed class responsible for handling Firebase events from `FirebaseMessagingService` to `.service.NotificationsService` on Android. ([#10558](https://github.com/expo/expo/pull/10558) by [@sjchmiela](https://github.com/sjchmiela))
 
   > Note that this change most probably will not affect you — it only affects projects that override `FirebaseMessagingService` to implement some custom handling logic.
-
```

Comment:
undo- does prettier do this auto for you? Mine doesn't seem to

Reply:
I don't think I changed that, I just ran `et add-changelog ...`